src/reset.css

```diff
@@ -0,0 +1,14 @@
+* {
+  box-sizing: border-box;
+}
+
+body,
+h2,
+p {
+  margin: 0;
+}
+
+ul {
+  margin: 0;
+  /* list-style: none; */
```

Comment:
Remove comment. Try it write obvious code that explain all we need without comments)

Reply:
not fixed